bin/nginx_release_downloads.sh

```diff
@@ -14,22 +14,28 @@
 
 downloads=https://nginx.org/download
 
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  SED='gsed'
+else
+  SED='sed'
+fi
+
 if [ $# -gt 1 ]; then
     >&2 printf 'At most one argument supported, but %d were specified.\n' $#
     exit 1
 fi
 
 filter() {
     if [ $# -eq 1 ]; then
-        grep "^$(echo "$1" | sed 's/\./\\./g')[. ]" | tail -1 | awk '{print $2}'
+        grep "^$(echo "$1" | $SED 's/\./\\./g')[. ]" | tail -1 | awk '{print $2}'
```

Comment:
`shellcheck` will want `"$SED"` here and below.

Reply:
Shellcheck (0.9.0 on my machine) doesn't complain about this since we explicitly set `$SED` to one of two strings without whitespace earlier in the script.